iib/workers/tasks/build_merge_index_image.py

```diff
@@ -272,9 +275,11 @@ def handle_merge_request(
             file_mode=(stat.S_IRUSR | stat.S_IWUSR | stat.S_IRGRP | stat.S_IWGRP),
         )
 
-    output_pull_spec = _create_and_push_manifest_list(request_id, prebuild_info['arches'])
+    output_pull_specs = _create_and_push_manifest_list(
```

Comment:
Here is plural and in the file `build.py` is just `output_pull_spec` instead of `spec**s**`

Reply:
it seems there is a discrepancy. THe function `_create_and_push_manifest_list` returns `return output_pull_specs[0]`. You are using it differently here.